### PR TITLE
feat: add NOTAMs

### PR DESCRIPTION
**Pull request by ZigTag**
## Summary of changes
Adds NOTAMs blog-style feature for announcements and updates

## Desktop
image
image
image

## Mobile
image

image

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,2 +1,5 @@
 // eslint-disable-next-line strict
```
Comment (NathanInnes) [inline next.config.js:1]:
```suggestion
'use strict';
```

**Review by NathanInnes — COMMENTED** [review]
Please look into ts-ignores and try and find a solution

**Comment by NathanInnes** [comment]
In many places the UI needs some improvements. I cannot comment on all but, overall feeling of the NOTAM area feels boring. We can try and get some figma concepts going to improve the layout.

**Comment by ErickSharp** [comment]
NOTE: I also fed in the metaImage and metaAlt tags into each blog page so now the hero has the same image as the one on the post card. Also, here is what the page would look like with both announcement and update posts on them:

image
image
image

**Review by NathanInnes — APPROVED** [review]
Looking good to release. At least 1 more review should be aquired before merge

File: posts/welcome.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (NathanInnes) [inline posts/welcome.md:1]: Real content should be placed in in preperation for merge.

File: tailwind.config.js [hunk 3]
```diff
@@ -38,10 +40,43 @@ module.exports = {
                     DEFAULT: 'hsl(227,58%,65%)',
                     dark: 'hsl(227,32%,49%)',
                 },
+                cool: { DEFAULT: 'hsl(219, 100%, 95%)' },
```
Comment (NathanInnes) [inline tailwind.config.js:43]: I would give a better name to 'cool'

Comment (ErickSharp) [reply tailwind.config.js:43]: its cool though? 🥶😎

**Review by NathanInnes — APPROVED** [review]
I am happy with the current version of this to be released.

**Review by ErickSharp — APPROVED** [review]
LGTM :D

**Pull request merged**